src/scripts/converter.py

```diff
@@ -0,0 +1,20 @@
+#! /usr/bin/env python
+import mdsynthesis as mds
+import json
+from os import path
+from glob import glob
+
+def convert(folder):
+    json_file = glob(path.join(folder, 'Sim*'))[0]
+    with open(json_file) as fh:
+        old = json.load(fh)
+    mds.Sim(folder, categories=old['categories'], tags=old['tags'])
```

Comment:
We also need to convert the atomgroups stored in the sim object. @dotsdl what else is stored in a  sim?